reference/7.1/Microsoft.PowerShell.Utility/Write-Host.md

```diff
@@ -24,18 +24,25 @@ Write-Host [[-Object] <Object>] [-NoNewline] [-Separator <Object>] [-ForegroundC
 
 ## DESCRIPTION
 
-The `Write-Host` cmdlet customizes output.
-You can specify the color of text by using the `ForegroundColor` parameter, and you can specify the background color by using the `BackgroundColor` parameter.
-The Separator parameter lets you specify a string to use to separate displayed objects.
-The particular result depends on the program that is hosting PowerShell.
+The `Write-Host` cmdlet's primary purpose is to produce for-(host)-display-only output, such as
+printing colored text like when prompting the user for input in conjunction with [Read-Host](Read-Host.md).
+`Write-Host` uses the .NET method, [ToString()](/dotnet/api/system.object.tostring) to write the
```

Comment:
Nice, except fo this pet peeve alert: the use of comma after ".NET method" would imply that only _one_ such method exists overall; hence: :uses the [ToString()](/dotnet/api/system.object.tostring) .NET method"

Also, it looks this enhanced description is only in the v7.1 document, but the info equally applies to all previous versions.